### PR TITLE
Landing Page Updates

### PR DESCRIPTION
**Pull request by shahmonanshi**

### CODE REVIEW

**Review by colearendt — APPROVED** [review]
I haven't tested this - in PRs like this (where it takes a good bit of work to "test" it - it can often be good to "screen shot" the latest for review 😄 

Looks great! My only feedback is that I would honestly lean towards burning all the files in the `img` directory, and putting your images there 😄 No need to keep all the old stuff around! We can always bring it back from the git history if needed.

File: index.html [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
   pre:not([class]) {
-    background-color: white;
+    background-color: #ffffff;
   }
 </style>
 
 
 
-<style type="text/css">
+<style type="text/css" href="air.css">
```
Comment (colearendt) [inline index.html:25]: Does this still get read with the `href`?

**Comment by colearendt** [comment]
(Oh nvm, totally missed that this is not the latest haha)

**Comment by shahmonanshi** [comment]
Oh totally, I can share a screenshot here. Theres also a VS code extension (live server) that can get you a localhost set up with a click of a button! 
image

**Pull request merged**